linux_os/guide/system/software/integrity/fips/grub2_enable_fips_mode/oval/shared.xml

```diff
@@ -4,8 +4,6 @@
       <title>Enable FIPS Mode in GRUB2</title>
       <affected family="unix">
         <platform>Red Hat Enterprise Linux 7</platform>
```

Comment:
Still applicable to rhel8. Please undo the changes.

Reply:
No, it is not.

Reply:
Process for enabling FIPS mode in RHEL8 and Fedora have changed.